src/components/password-strength-field/password-strength.html

```diff
@@ -0,0 +1,30 @@
+<div>
+    <meter max="4" low ="3" optimum="4" [value]="passwordStrengthScore"></meter>
+    <p class="password-strength-text">{{ passwordStrengthScoreText }}</p>
+    <div class="content-div">
+        <button ion-button class="toggle-advanced-button" (click)="togglePasswordAdvanced()">
+                {{ 'password-strength-component.toggle-advanced' | translate }}
+        </button>
+    </div>
+    <div *ngIf="showPasswordAdvanced">
+        <div *ngIf="passwordFeedbackWarning.length > 0">
+            <p class="password-advanced-feedback">{{ 'password-strength-component.feedback' | translate }}</p>
+            <ul class="password-feedback-warning-list">
+                <li *ngFor="let feedbackWarning of passwordFeedbackWarning" class="password-feedback-warning-list-item">{{ feedbackWarning }}</li>
+            </ul>
+        </div>
+        <p id="time-to-crack">{{ 'password-strength-component.time-to-crack' | translate }}</p>
```

Comment:
Does this really need to use an id and not a class? Using a class makes this component more reusable and makes it easier to use it multiple times on a single page.

Even if not using it multiple times on a single page the way Ionic pages are structured (e.g. simply overlayed on top of each other) makes it quite easy to have this component multiple times in your DOM.